components/button/README.md

```diff
@@ -0,0 +1,8 @@
+This component is used to implement dang sweet buttons.
+
+#### Props
+
+The following props are used to control the display of the component. Any additional props will be passed to the rendered `<a />` or `<button />` element. The presence of a `href` prop determines whether an anchor element is rendered instead of a button.
+
+* `isPrimary`: (bool) whether the button is styled as a primary button.
+* `href`: (string) if this property is added, it will use an `a` rather than a `button` element.
```

Comment:
I might be open to reconsidering that if it serves good purpose for documentation.